dev/com.ibm.websphere.appserver.features/visibility/public/xmlBinding-3.0/io.openliberty.xmlBinding-3.0.feature

```diff
@@ -12,16 +12,17 @@ IBM-API-Package: \
   jakarta.xml.bind.attachment;  type="spec", \
   jakarta.xml.bind.helpers;  type="spec", \
   jakarta.xml.bind.util;  type="spec"
-IBM-ShortName: jaxb-3.0
+IBM-ShortName: xmlBinding-3.0
 IBM-Process-Types: client, \
  server
 Subsystem-Name: Jakarta XML Bindings 3.0
 -features=\
   com.ibm.websphere.appserver.classloading-1.0, \
   com.ibm.websphere.appserver.eeCompatible-9.0, \
-  io.openliberty.jakarta.jaxb-3.0
+  io.openliberty.jakarta.xmlBinding-3.0
 -bundles=\
-  io.openliberty.jaxb.3.0.internal.tools
+  io.openliberty.xmlBinding.3.0.internal.tools
```

Comment:
I think now with the other changes you made you can do:
```
-files=\
  bin/xmlBinding/xjc.bat, \
  bin/xmlBinding/tools/ws-schemagen.jar, \
  bin/xmlBinding/schemagen; ibm.executable:=true; ibm.file.encoding:=ebcdic, \
  bin/xmlBinding/xjc; ibm.executable:=true; ibm.file.encoding:=ebcdic, \
  bin/xmlBinding/tools/ws-xjc.jar, \
  bin/xmlBinding/schemagen.bat
```

Reply:
This assumes that the files are created in that location. After you ran your local build do you see the files I am referencing here?